CHARTER.adoc

```diff
@@ -46,15 +57,15 @@ The TODO Group will host Working Groups for members to contribute to that are in
  ... Meetings will occur at least once per quarter.
  ... Details of meeting mechanisms and instructions for attendance will be published reasonably in advance to all members in good standing.
  .. Communication mechanisms
- ... All public communication not during meetings will take place over the public group mailing list(s) accessible to members and non-members alike.
- ... All private communication not during meetings will take place over the private group mailing list for members only.
+ ... All public communication not during meetings will take place over the public group mailing list(s), real-time chat, and GitHub and will be accessible to members and non-members alike.
+ ... All private communication not during meetings will take place over the private group mailing list, real-time chat, and GitHub and access will be limited to members only.
  .. Level of confidentiality
  ... All communication during meetings will be under+++<u>+++ Chatham House Rules+++</u>++++++<u>+++.+++</u>+++
- ... All communication on the private mailing list will be restricted to members only and not permitted for public distribution.
- ... All communication on the public mailing list will be public.
+ ... All private communication via all media as defined above will be restricted to members only and not permitted for public distribution.
+ ... All communication via all media as defined above will be public.
  . *Antitrust Guidelines*
  .. All members shall abide by The Linux Foundation Antitrust Policy available at: https://www.linuxfoundation.org/antitrust-policy/.
- .. All members shall encourage open participation from any organization able to meet the membership requirements, regardless of competitive interests. Put another way, the TODO Group shall not seek to exclude members based on any criteria, requirements or reasons other than those used for all members.
+ .. All members may encourage open participation from any organization able to meet the membership requirements, regardless of competitive interests. Put another way, the TODO Group shall not seek to exclude members based on any criteria, requirements, or reasons other than those used for all members.
  . *General Rules and Operations*.
 The TODO Group shall be conducted so as to:
```

Comment:
```suggestion
The TODO Group General Members and broader community shall be conducted so as to:
```

Reply:
I would like to best understand the exclusion criteria. I have seen organization's that appear to meet our criteria be told they cannot join TODO. Can we unpack this a bit in SC session?